integration/app/test-demo/src/containers/MeetingForm.tsx

```diff
@@ -24,8 +25,7 @@ const MeetingForm: React.FC = () => {
     try {
       const { JoinInfo } = await fetchMeeting(meetingId, name, 'us-east-1');
       await meetingManager.join({
-        meetingInfo: JoinInfo.Meeting,
-        attendeeInfo: JoinInfo.Attendee,
+        meetingSessionConfiguration: new MeetingSessionConfiguration(JoinInfo.Meeting, JoinInfo.Attendee),
```

Comment:
This `meetingSessionConfiguration` needs to be outside right with the signature change?

Reply:
Yeah `meetingSessionConfiguration` itself is an argument now.

Reply:
I was making these changes while comments came. Lol.